### DISCUSSION TITLE
Start a single node elastic cluster with Docker Compose | ashish.one

### DISCUSSION THREAD

**Discussion by giscus** [ashishtiwari1993/ashish.one#20] [General]
# Start a single node elastic cluster with Docker Compose | ashish.one

Introduction In this gist, we will quickly try to spin Elastic stacks with Docker containers. We are going to use docker-compose. You can learn more about Docker & Docker Compose, Which will help you to understand the flow.
Prerequisite Tested on the below configuration.
docker:Docker version 20.10.16, build aa7e414 docker-compose:Docker version 20.10.16, build aa7e414 Cluster This setup will include
Elasticsearch Kibana Logstash APM Setup Clone repo:
git clone https://github.com/ashishtiwari1993/elastic-docker.git cd elastic-docker Make changes in .

https://ashish.one/blogs/elastic-docker-compose/

**Comment by andrewbain** [comment]
HI this works great locally any tips on getting this working on EC2 cant connect to server

**Reply by ashishtiwari1993** [reply]
Please make sure to open port on ec2 machine if you trying to access from outside. https://aws.amazon.com/premiumsupport/knowledge-center/connect-http-https-ec2/

Also check elastic instances are up and working on your same machine (ec2) by hitting curl command locally
